### ISSUE TITLE
The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells

### ISSUE THREAD

**Issue by ami-day** [ebi-ait/hca-ebi-wrangler-central#588]
### Project short name:
cellFateDynamics
### Primary Wrangler:
@ipediez 
### Associated files

* Google Drive: https://drive.google.com/drive/folders/1sElWkqie-5kR6LjbOeYPS_FAp49wCMQG
* Ingest: https://contribute.data.humancellatlas.org/projects/detail?uuid=1eb69a39-b5b2-41ec-afae-5fe37f272756&tab=project
### Published study links

* Paper: https://doi.org/10.1038/nbt.2859

* Accessioned data: GSE52529

### Key Events

- [x] Convert published metadata to HCA spreadsheet
- [x] Manually curate dataset to meet HCA metadata standard
- [x] Collect any matrix and cell-type annotation files
- [x] Upload sheet to validate metadata
- [x] Check linking using ingest graph validator
- [x] Transfer raw files to ingest to validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [x] Submit dataset to Production 
- [x] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by idazucchi** [comment]
@ESapenaVentura secondary reviewer

**Comment by ipediez** [comment]
it has a graph valid submission, but has ben unlinked to the project: https://contribute.data.humancellatlas.org/submissions/detail?uuid=49bce4e2-2078-433e-987e-af7d11b97bb3

**Comment by ESapenaVentura** [comment]
Overall LGTM! I have a couple of comments:

## Cell line
- Cell morphology: This is about the morphology of the culture, if I remember my biology well myoblasts will probably form single layer colonies?

## Cell suspension
- I would add MEM-alpha as the growth medium for the cell suspensions
- In timecourse relevance there is a typo, it should be “were”
- Tempted to delete the cell count from the bulk suspensions, the browser will add that to the count total and they’re not really single cells. What do you think?

## Analysis protocol
- I think it’s missing the alignment etc steps, I found some extra info in geo (e.g. https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1268960)

**Comment by ipediez** [comment]
I've expanded the cell morphology description, MEM-alpha as growth medium for cell suspensions and the alignment step as you suggested. I have also added Cuffdif 2.2 as the computational method for the FPKM normalization protocol and corrected the typo at the timecourse relevance. Finally, I've deleted the cell count from the bulk suspensions and added it to their description, just to make sure that the information is not lost.
